core/scss/utilities/variables/core/_modular-spacing.scss

```diff
@@ -6,21 +6,21 @@
 // The amount of spacing around elements at certain breakpoints.
 //
 // Markup: $modular-spacing: (
-//   'xs': 32px,
-//   'sm': 32px,
-//   'md': 36px,
-//   'lg': 36px,
-//   'xl': 36px,
-//   '2xl': 38px,
+//   'xs': 20px,
+//   'sm': 30px,
+//   'md': 50px,
+//   'lg': 80px,
+//   'xl': 100px,
+//   '2xl': 150px,
 // ) !default;
 //
 // Styleguide: Variables.Core.modular-spacing
 //
 $modular-spacing: (
```

Comment:
The centered container is working great now. There's just one thing - I believe the "margin" is a new variable that's different and not related to the  "modular spacing" variable. Added screenshot from Figma with the "modular spacing" values. Perhaps create a new variable for the margins?

![image](https://user-images.githubusercontent.com/42749717/51767066-21260f80-2091-11e9-988e-cfe2eb858731.png)

Reply:
So the margins are only for the outside edge of a container to the browser window? We have three spacing models? 

Do we have rules on when to apply each documented somewhere? I am confused by this.

Reply:
Yup, that's my understanding from talking to John. The margins are completely on their own and aren't really used anywhere else. If I remember correctly, John and Kerri looked at some sites and adjusted the margin numbers until things looked the best for each break point.

I do agree that having 3 different spacing models can be confusing. Perhaps we could at least rethink the gutters in terms of modular spacing, eg, a XS gutter would be modular-scale(1) or something.   Talk more in the Decanter meeting?

Reply:
As for when to apply each - I don't think that's documented, but here's my understanding of it (from talking to John and working on homesite):
1. margins - only for outside edge of centered container to browser window
2. gutters - grid gutters, so naturally also become spacing between items that follows the "x numbers per row" model eg, cards.
3. modular spacing - a unitless spacing system used for padding/margins within and between some items, eg, for things like vertical rhythm, card padding etc.

However, I believe that we can rewrite 1 and 2 in terms of 3.

Reply:
Ok, this will make a good topic for Wednesday. I have refactored to put the screen edge margins into their own file. Generally, I am ok with these three spacing rules as long as we document them well.

Reply:
Awesome - thanks, and I agree, we need to document these rules.